bootstrap

```diff
@@ -191,6 +175,8 @@ while (true) {
       return strlen($header);
     }
 
+    $name = strtolower($name);
```

Comment:
I'd rather not lose the fidelity of the header name as it passes through the layer. If the API Gateway or ALB can pass it with a specific case, let's leave it as-is.